mesh/mesh.go

```diff
@@ -140,6 +140,11 @@ func (msh *Mesh) recoverFromDB(latest types.LayerID) {
 		log.Stringer("processed", msh.ProcessedLayer()))
 }
 
+// Tortoise returns the tortoise instance.
+func (mesh *Mesh) Tortoise() system.Tortoise {
+	return mesh.trtl
+}
```

Comment:
This hides the dependency of the `Syncer` through the dependency on the `Mesh`.

Would it make sense to instead make the `Syncer` depend on tortoise directly?

Reply:
you are right, i will change that